includes/fields/class-evf-field-number.php

```diff
@@ -36,13 +36,89 @@ public function __construct() {
 					'placeholder',
 					'label_hide',
 					'css',
+					'maximum_number',
+					'minimum_number',
+
 				),
 			),
 		);
 
 		parent::__construct();
 	}
 
+
+
+
+	/**
+	 * Minimum number field option
+	 *
+	 * @since      1.4.5
+	 *
+	 * @param array $field
+	 */
+	public function minimum_number( $field ) {
+		$lbl  = $this->field_element(
+			'label',
+			$field,
+			array(
+				'slug'    => 'minimum_number',
+				'value'   => esc_html__( 'Minimum Number', 'everest-forms-pro' ),
+				'tooltip' => sprintf( esc_html__( 'Enter Minimum Number Allowed.', 'everest-forms-pro' ) ),
```

Comment:
This sentence should be clear and appropriate so that users can understand easily.